content/zh-cn/glossary/vocabulary.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "名词解释"
+draft: true
+---
+IoT Atlas中出现的名词及定义
+<!--more-->
+
+### 执行器(Actuator)
+> 执行器用来对机械进行移动或控制
+
+### 命令消息(Command Message)
+> 一个[消息](#message)描述了设备应该采取的动作，并且一旦动作完成，设备就知道应该在另一个主题上进行确认。
+
+### 设备(Device)
+> 是具有足够的供电和计算资源的实体，作为IoT解决方案的一部分。大多数情况下设备连接[传感器](#sensor)或[执行器](#actuator)。
+
+### 设备数据密度（Device Data Density）
+> 一个设备的数据密度是：`密度 ＝ 采样频率 * 样本大小 * 报告频率`
+
+### 端点(Endpoint)
+> 待定
+
+### 有限生产(Limited Production)
+
+> 验证在规模化场景下用户对项目的边缘和云软件的程度，通过蓝图满足在大规模生产环境下主要的用户用例
+
+### 消息(Message)
+> IoT解决方案中的通信单元，经常包含感测或命令相关的数据
+
+### 消息代理(Message Broker)
+> IoT解决方案中负责管理授权发布者和主题订阅者之间消息路由的组件
+
+### 消息主题(Message Topic)
+> 消息主题，即*主题(topic)*，是与消息相关联的主题(subject)。主题经常被用来从发布者路由消息到订阅者。在这个上下文环境中，消息主题应该直接对应到发布/订阅概念中的主题(subject)，并类似于用来描述请求/响应和REST端点的相对URL(partial URL)的概念。
+
+### 有序存储(Ordered Store)
+> 是在IoT解决方案中确保所有记录都按时序进行存储的数据存储或微服务。
+
+### 试点(Pilot)
+> 通过测试一小部分用户用例、设备配置和/或者最终用户功能，识别业务成果、软件架构和/或创建设备蓝图
+
+### 生产(Production)
+> 大规模生产设备，并将IoT产品推向市场
+
+### 原型(Prototype)
+> 以教学为目的，获取客户管理层批准开始试点阶段
+
+### 规则(Rule)
+> 由规则引擎执行的逻辑和动作的单一集合
+
+### 规则引擎(Rule Engine)
+> 详见维基百科([Business rules engine](https://en.wikipedia.org/wiki/Business_rules_engine))
```

Comment:
Please ensure that these Wikipedia URLs are pointing to the Simplified Chinese version on Wikipedia (if that version exists).